### DISCUSSION TITLE
Info

### DISCUSSION THREAD

**Discussion by ritchiemarkov** [wagtail/wagtail#10655] [General]
Hello everyone! Before start my journey with wag I'm trying to understand if it the right tool for me and I'm sure you can clarify some of my doubts.

I'm currently using WordPress for my blog with bootstrap based theme.

Question 1) can tailwind be implemented in wag?
Question 2) can external javascript libraries can be implemented in pluginish way? Let's take a swiper.js for example and make some advanced carousels?

It's not clear for me how external things are managed. I see it's has a great text editor but I don't understand how to implement external js libraries.

Can also wag be installed on WordPress host? I mean it's running on python as I understood well but should I have ssh access (node. Js like)?

I'm sorry for dumb questions btw.

**Comment by Scotchester** [comment]
Hey phrizm,

The short answer to your questions is "Yes!" Unlike WordPress, Wagtail does not provide any front end for your website; it's up to you to write the HTML, CSS, and JavaScript to display the content you manage in it. Actually, it does provide default markup for rendering certain things, like rich text fields or table blocks, for example, but the overall front end that contains those things is up to you. So, yes, you can use Tailwind, Swiper, or any other front-end libraries you like.

If the WordPress host you're using also supports Python, then in theory, it could be installed there. SSH access is not necessarily required, depending on what they offer, but could come in handy.

**Reply by ritchiemarkov** [reply]
Thank you so much for your kind reply.

I will deep dive into, the project seems to be really promising and from my (limited experience) point of view really powerfull.

Once again ty!
